Vagrantfile

```diff
@@ -9,62 +29,23 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # options are documented and commented below. For a complete reference,
   # please see the online documentation at vagrantup.com.
 
+  #calculate memory allocation
   # Every Vagrant virtual environment requires a box to build off of.
-  config.vm.define "frontend" , primary: true do |fe|
-    fe.vm.box = 'M0ses/openSUSE-Leap-42.1-minimal'
+  config.vm.define "development" , primary: true do |fe|
+    fe.vm.box = 'opensuse/openSUSE-42.1-x86_64'
     # Provision the box with a simple shell script
-    fe.vm.provision :shell, inline: '/vagrant/contrib/bootstrap_frontend.sh'
-    fe.vm.provision :shell, inline: 'mount /vagrant/src/api/tmp', run: "always"
+    fe.vm.provision :shell, inline: '/vagrant/contrib/bootstrap.sh'
+    #fe.vm.provision :shell, inline: 'mount /vagrant/src/api/tmp', run: "always"
```

Comment:
I commented it out during the development to test something. I forgot to put it in again. I also had to call the _prepare_bound_directory function with setup_data_dir() (was setup_obs_backend)

It was missed during reconstruction.

I already put it back in. Will rebase...